contracts/ExpectedRate.sol

```diff
@@ -0,0 +1,41 @@
+pragma solidity ^0.4.18;
+
+
+import "./ERC20Interface.sol";
+import "./KyberNetwork.sol";
+import "./PermissionGroups.sol";
+
+
+interface ExpectedRateInterface {
+    function getExpectedRate ( ERC20 source, ERC20 dest, uint destQuantity ) public view
```

Comment:
user will never know dest qty. it should be source qty.

Reply:
done
@yaronvel  please merge